### PR TITLE
Added write function to inventory script and redfish utilities

### PR DESCRIPTION
**Pull request by jbrawdy**
I added a function to the inventory file in redfish_utilities to write the inventory to an excel spreadsheet. Then added a new command line argument to give the option of writing the gathered inventory to an excel spreadsheet.

### CODE REVIEW

File: redfish_utilities/inventory.py [hunk 1]
```diff
@@ -284,3 +286,49 @@ def print_system_inventory( inventory_list, details = False, skip_absent = False
...
+def write_system_inventory( inventory_list):
+    """
+    Write the system inventory list into a spreadsheet
+
+    Args:
+        inventory_list: The inventory list to write to an Excel spreadsheet
+    """
+
+    # Excel workbook to save data extracted and parsed
+    workbook = xlsxwriter.Workbook("./Device_Inventory.xlsx")
```
Comment (mraineri) [inline redfish_utilities/inventory.py:300]: Could we make this an argument for the function (and have it default to "./Device_Inventory.xlsx" if not provided)?

Comment (jbrawdy) [reply redfish_utilities/inventory.py:300]: That should be possible. I'll see what I can come up with.

Comment (jbrawdy) [reply redfish_utilities/inventory.py]: I have updated the pull request. Is what I added what you had in mind?

**Comment by mraineri** [comment]
Approved 7/9

**Comment by jbrawdy** [comment]
Thanks for the merge! What does approved 7/9 mean?

**Comment by mraineri** [comment]
We have group meetings each week where we go over open issues and approve pull requests. We had such a meeting today and others agreed the change looks good to be merged.

**Pull request merged**